app/api/v1/endpoints/users.py

```
@@ -0,0 +1,34 @@
+    skip: int = 0,
+    limit: int = 100,
+    db: Session = Depends(get_db)
+):
+    logger.info(f"Fetching users with skip={skip}, limit={limit}")
+    users = await user_crud.get_users(db, skip=skip, limit=limit)
+    return users
+
+@router.post("/", response_model=user_schema.UserCreate)
+async def create_user(
+    user: user_schema.UserCreate,
+    db: Session = Depends(get_db)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)